### DISCUSSION TITLE
Task 02 - Lab 03.02 Time Series Model

### DISCUSSION THREAD

**Discussion by Alejandro-A-DS** [rramosp/ai4eng.v1#62] [03 - GENERACION DE MODELOS]
Un saludo.

Estoy realizando la actividad 2 (donde toca aplicar manualmente la fórmula de regresión lineal) y definí mi función que funciona, sin embargo no he podido saber cómo hacer esa función en una sola línea de python como lo pide el reto.
¿Alguien me podría dar una pista de cómo podría hacerse? En mi función usé un for loop, y un par de operaciones en un par de líneas de código.

Muchas gracias de antemano.

**Comment by rramosp** [comment]
hola alejandro, prueba a (1) selecciona las columnas de `td` que necesitas, (2) obtén un array `numpy` con el `.values` del dataframe resultante; y (3) usa `.dot` mas una suma con  `w` para obtener el resultado

**Reply by Alejandro-A-DS** [reply]
Sí señor, el `.dot` era la función que me hacía falta tener en cuenta. Muchas gracias profesor.
